common.gypi

```diff
@@ -148,6 +148,17 @@
             'cflags': [ '-gxcoff' ],
             'ldflags': [ '-Wl,-bbigtoc' ],
             'conditions': [
+              ['target_arch=="ppc64"', {
+                'ldflags': [
+                  '-Wl,-blibpath:/usr/lib:/lib:'
+                    '/opt/freeware/lib/pthread/ppc64'
+                ],
+              }],
+              ['target_arch=="ppc"', {
```

Comment:
Does it make sense to make these if/else instead of two ifs?
Also do we have test coverage for both AIX/ppc and AIX/ppc64?

Reply:
> 
> 
> Does it make sense to make these if/else instead of two ifs?

Possibly. Currently these are consistent with the existing equivalents in the the `Release` section further down in the same file. I'd prefer to be consistent, so let me know if you want both changed (or we can do that in another PR).

> Also do we have test coverage for both AIX/ppc and AIX/ppc64?

We've never supported AIX/ppc in the community builds and thus have no CI coverage (neither build nor test). AFAIK we also don't have coverage now internally at IBM since [last year](https://developer.ibm.com/node/2018/04/25/going-open-deprecating-the-ibm-sdk-for-node-js-in-favour-of-the-community-builds/). We could perhaps open a separate discussion about removing AIX/ppc stuff from the build files, but that should be a separate explicit issue.

Reply:
@richardlau thanks!